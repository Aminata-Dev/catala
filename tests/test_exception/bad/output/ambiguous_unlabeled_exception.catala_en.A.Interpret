[ERROR] This exception can refer to several definitions. Try using labels to disambiguate

Ambiguous exception
  --> tests/test_exception/bad/ambiguous_unlabeled_exception.catala_en
   | 
10 |   definition x equals 1
   |                       
11 | 
   | 
12 |   exception
   |   ^^^^^^^^^
13 |   definition x equals 2
   |   ^^^^^^^^^^^^^^^^^^^^^^
   + Test

Candidate definition
  --> tests/test_exception/bad/ambiguous_unlabeled_exception.catala_en
   | 
10 |   definition x equals 1
   |              ^
   + Test

Candidate definition
 --> tests/test_exception/bad/ambiguous_unlabeled_exception.catala_en
  | 
8 |   definition x equals 0
  |              ^
  + Test
