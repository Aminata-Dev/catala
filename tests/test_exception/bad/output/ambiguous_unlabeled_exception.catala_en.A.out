[ERROR] This exception can refer to several definitions. Try using labels to disambiguate

Ambiguous exception
  --> test_exception/bad/ambiguous_unlabeled_exception.catala_en
   | 
10 |   definition x equals 1
   |                       
11 | 
   | 
12 |   exception
   |   ^^^^^^^^^
13 |   definition x equals 2
   |   ^^^^^^^^^^^^^^^^^^^^^^
   + Test

Candidate definition
  --> test_exception/bad/ambiguous_unlabeled_exception.catala_en
   | 
10 |   definition x equals 1
   |              ^
   + Test

Candidate definition
 --> test_exception/bad/ambiguous_unlabeled_exception.catala_en
  | 
8 |   definition x equals 0
  |              ^
  + Test
